container.go

```diff
@@ -49,6 +49,8 @@ type Container interface {
 	NetworkAliases(context.Context) (map[string][]string, error) // get container network aliases for a network
 	Exec(ctx context.Context, cmd []string) (int, error)
 	ContainerIP(context.Context) (string, error) // get container ip
+	CopyToContainer(ctx context.Context, containerPath string, content io.Reader) error
```

Comment:
Shouldn't it be called "CopyFromContaner" ?

Reply:
@joe-getcouragenow My use case is a little different than yours. I need to copy a single file to the container. CopyToContainer copies a compacted file to the container (using io.Reader), I then implemented CopyFileToContainer as well to wrap and allow to copy only one file to the container, using CopyToContainer.

Does that make more sense to you?

Reply:
Yep it does :)

Thanks.

I will try to do the copy form Container --> Host